ethereum/eth/src/main/java/tech/pegasys/pantheon/ethereum/eth/transactions/TransactionPoolFactory.java

```diff
@@ -20,9 +20,12 @@
 import tech.pegasys.pantheon.metrics.MetricsSystem;
 
 import java.time.Clock;
+import java.util.concurrent.TimeUnit;
 
 public class TransactionPoolFactory {
 
+  private static final long TRANSACTION_EVICTION_INTERVAL_MS = TimeUnit.MINUTES.toMillis(1);
```

Comment:
The default needs to be much longer than 1 minute.  Probably more like 12 hours since it's not currently configurable.